test/selenium/run-tests

```diff
@@ -32,6 +32,8 @@ CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
 if CURRENT_DIR not in sys.path:
     sys.path.insert(1, CURRENT_DIR)
 
+TEST_TIMEOUT = 600
```

Comment:
can you please prefix this commit with "test:", and add a "Closes #13125" to the commit message?

Reply:
yep, done. I hope that in proper format

Reply:
The Closes goes at the bottom of commit description, not the Subject line; but as the PR is now just one commit, I can fix that during squashing.